solidity/contracts/deposit/Deposit.sol

```diff
@@ -12,13 +12,24 @@ import {FeeRebateToken} from "../system/FeeRebateToken.sol";
 
 import "../system/DepositFactoryAuthority.sol";
 
-/// @title  Deposit.
-/// @notice This is the main contract for tBTC. It is the state machine
-/// that (through various libraries) handles bitcoin funding,
-/// bitcoin-spv proofs, redemption, liquidation, and fraud logic.
-/// @dev This is the execution context for libraries:
-/// `DepositFunding`, `DepositLiquidaton`, `DepositRedemption`,
-/// `DepositStates`, `DepositUtils`, `OutsourceDepositLogging`, and `TBTCConstants`.
+/// @title  tBTC Deposit
+/// @notice This is the main contract for tBTC. It is the state machine that
+///         (through various libraries) handles bitcoin funding, bitcoin-spv
+///         proofs, redemption, liquidation, and fraud logic.
+/// @dev This contract presents a public API that exposes the following
+///      libraries:
+///
+///       - `DepositFunding`
+///       - `DepositLiquidaton`
+///       - `DepositRedemption`,
+///       - `DepositStates`
+///       - `DepositUtils`
+///       - `OutsourceDepositLogging`
+///       - `TBTCConstants`
+///
+///      Where these libraries require deposit state, this contract's state
+///      variable `self` is used, a struct of type `DepositUtils.Deposit` that
```

Comment:
`self is used, a struct...` -> `self is used. self is a struct of type `DepositUtils.Deposit...`?